_platforms/flip.md

```diff
@@ -4,4 +4,11 @@ platform-name: FLIP
 platform-image: /assets/img/platforms/flip_logo.png
 ---
 
-FLIP information.
\ No newline at end of file
+FLIP, the Federated Learning Interoperability Platform, links data from multiple Trusts to enable AI at scale. It is composed of three core components:
```

Comment:
Can we get an additional sentence to say _why_ we want to/it's important to enable AI at scale. Just to be a bit more explicit about the motivation

Reply:
I've just added a (very long!) one-liner but let me know if it's off-base/needs to be trimmed down...